src/frontend/src/hooks/users.hooks.ts

```diff
@@ -247,3 +248,10 @@ export const useUserTasks = (userId: string) => {
     return data;
   });
 };
+
+export const useManyUserTasks = (userIds: string[]) => {
```

Comment:
Trying to get more documentation for our custom hooks - can you add a jsdoc comment here?